horace_core/sqw/@sqw/mask_pixels.m

```diff
@@ -1,24 +1,24 @@
-function wout = mask_pixels (win, mask_array)
+function wout = mask_pixels (win, keep_obj)
```

Comment:
`keep_obj` is a bad name. Maybe `pix_to_keep`, honestly `mask_array` was probably best

Reply:
modified according to review above

Reply:
`mask_array` assumes that you remove data where the mask is true. The code does opposite -- this is why the removing done.